zipline/finance/performance/tracker.py

```diff
@@ -77,7 +78,7 @@ class PerformanceTracker(object):
     """
     Tracks the performance of the algorithm.
     """
-    def __init__(self, sim_params, trading_calendar, env):
```

Comment:
We're using the `treasury_curves` attribute, but I believe that's now effectively unused.  @ehebert can you confirm that nothing should be using the treasury curves anymore?